docs/csharp/language-reference/compiler-messages/cs0108.md

```diff
@@ -12,33 +12,37 @@ ms.assetid: 04330ed2-80d5-4bf2-b0c1-a0c2bec03074
 
 'member1' hides inherited member 'member2'. Use the new keyword if hiding was intended.
 
- A variable was declared with the same name as a variable in a base class. However, the [new](../keywords/new-modifier.md) keyword was not used. This warning informs you that you should use **new**; the variable is declared as if **new** had been used in the declaration.
+A member was declared with the same name as a member in a base class.
+However, the [new modifier](../keywords/new-modifier.md) was not used. 
```

Comment:
remove trailing space:

```suggestion
However, the [new modifier](../keywords/new-modifier.md) was not used.
```

Reply:
Yikes. This is already the second time in a row, I'll keep a closer eye to this in the future.
Thanks Bill!